bin/chat-fastapi.py

```diff
@@ -17,6 +18,20 @@
 CLOUDFLARE_SECRET_KEY = os.getenv("CLOUDFLARE_SECRET_KEY")
 CLOUDFLARE_SITE_KEY = os.getenv("CLOUDFLARE_SITE_KEY")
 
+ERROR_PAGE_TEMPLATE = Template(
+    f"""
+<html>
+    <body>
+        <h1>$error_title</h1>
+        <p>If you believe this to be in error, please contact the maintainers to report an issue.</p>
+        <form action="{CHAINLIT_URI}" method="get">
+            <button type="submit">Try again</button>
```

Comment:
Looking for feedback on this message. Should we include an email or link?

Reply:
For now it could be help@reactome.org